### PR TITLE
Change hover on link to white.

### LINKED ISSUE #94: The hover on menu link is blue on certain sub-pages
The main menu on library sub-pages, like /dewey/about, show #23527c on hover, unlike all other types of pages, it seems, which properly, show #fff.

### PR DESCRIPTION
**Pull request by PBruk**
Tagging @frrrances for code-review: resolves #94.

### CODE REVIEW

File: css/partials/_typography.scss [hunk 1]
```diff
@@ -42,6 +42,7 @@ a {
 	&.no-underline {
 		&:hover {
 			text-decoration: none;
+			color: #fff;
```
Comment (frrrances) [inline css/partials/_typography.scss]: This isn't the right place to put this color - it should probably be in _header.scss where the background color is defined as a dark color so we don't end up with white on white.

File: css/partials/_menu.scss [hunk 2]
```diff
@@ -15,8 +15,9 @@
     width: 100%;
     a {
       width: 100%;
-      &.hover {
-	      color: #fff;
+      &.hover,
```
Comment (frrrances) [inline css/partials/_menu.scss]: looks like an extra comma here

Comment (PBruk) [inline css/partials/_menu.scss]: Actually, if you notice, that period should be a colon, eh? And maybe that is why the hover on subpages wasn't working?

**Comment by frrrances** [comment]
👍

**Pull request merged**